index.js

```diff
@@ -23,9 +23,8 @@ module.exports = {
       let urlToVisit = decodeURIComponent(req.query.url);
       let parsed = url.parse(urlToVisit, true);
 
-      let headers = Object.assign(req.headers, {
-        host: 'ember-cli-fastboot-testing.localhost'
-      });
+      let headers = Object.assign(req.headers, req.query.headers)
+      headers.host = 'ember-cli-fastboot-testing.localhost';
```

Comment:
We shouldn't overwrite the host header if the user passed it in.

Reply:
Fixed!